src/HttpHandler/Guzzle7HttpHandler.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
One more nit: The new copyright should be

```suggestion
 * Copyright 2020 Google LLC
```

I'm sorry for not mentioning it before, since it's totally non-obvious!

Thanks again for all the help here, we all really appreciate it!

Reply:
I also could have checked in other files myself 🙈. Done! (And I enjoy the interaction as well!)